elated-egrets/bangazon-internal-site
# added edit view for departments

## DESCRIPTION
added edit view for departments

## How to pull down
Commands

```
git fetch --all
git checkout rm-make-departments-editable
```

## Steps to test

1. Start virtual environment
1. Make sure Django server is running
1. navigate to department details for a department
1. click the edit department link
1. give the department a new name
1. make sure the name has changed on the list view

## What you changed
added view for editing a department

## What was added to read me

## Final Checklist
Before merging.
1. Has each file changed been reviewed?
2. Has the readme been updated to reflect changes made?
3. Has a teammate approved the PR?

Then Ship It!
